pkg/project/version_bundle_aws.go

```diff
@@ -7,11 +7,11 @@ import (
 var versionBundleAWS = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
-			Component:   "cluster-operator",
-			Description: "TODO",
+			Component:   Name(),
+			Description: "Use Release.Revision in annotation for Helm 3 compatibility.",
```

Comment:
This isn't strictly necessary now that authorities have been deprecated. I'm okay with keeping it going until we actually remove `Changelog` from the operator though.

Reply:
Ack. Yes I just updated it because it was there. 

I'm glad we're rationalizing what we're doing with changelogs. 👍